generic_tracers/cobalt_glbl.F90

```diff
@@ -0,0 +1,1531 @@
+!> COBALT_glbl module consists of core parameters and vars 
+!! to be used by generic COBALT related modules
+!<----------------------------------------------------------------
+module cobalt_glbl
+  use field_manager_mod, only: fm_string_len, fm_path_name_len      
+  implicit none
+  public  
+
+  !The following variables for using this module
+  ! are overwritten by generic_tracer_nml namelist
+  logical :: do_nh3_diag                             !< logic for setting NH3 diagnostic tracer field 
+
+! Namelist Options
+
+  character(len=10) ::  co2_calc = 'ocmip2'          !< carbonate formalation options. Other option is 'mocsy'
+  logical :: do_14c             = .false.            !< If true, then simulate radiocarbon 
+  logical :: do_nh3_atm_ocean_exchange = .false.     ! If true, then do NH3 air-sea exchange 
+  !
+  logical :: do_vertfill_pre = .false.             !< Returns tracer arrays with sensible values
+  logical :: debug              = .false.          !< not use   
+  real    :: o2_min_nit= 0.01e-6                   !< Oxygen threshold for nitrification (mol O2 kg-1)
+  real    :: k_o2_nit  = 3.9e-6                    !< Oxygen half saturation constant for nitrification
+  real    :: irr_inhibit = 10.                     !< Irradiance inhibition term for nitrification (W m-2)
+  real    :: gamma_nitrif= 3.5e6                   !< Rate constant for nitrification (month-1)
+  real    :: k_nh3_nitrif= 3.1e-9                  !< NH3 half-saturation for nitrification (mol NH3 kg-1)  
+  real    :: imbalance_tolerance=1.0e-10           !< the tolerance for non-conservation in C,N,P,Sc,Fe
+
+  integer :: scheme_no3_nh4_lim = 2 !< Nitrate and ammonia limitation scheme options
+                                    !! 1-Frost and Franzen (1992)
+                                    !! 2-O'Neill
+
+  integer :: scheme_nitrif = 3      !< nitrification scheme options:
+                                    !! 1-default COBALT
+                                    !! 2-update with no temperature dependence
+                                    !! 3-update with temperature dependence
+
+namelist /generic_COBALT_nml/ do_14c, co2_calc, do_nh3_atm_ocean_exchange, scheme_nitrif, debug, &
```

Comment:
This won't affect the bug with the air-sea flux setting. I can send a separate PR on top of this one to finally resolve that.

Reply:
Sounds good. I will keep this one and it will fix by another PR.